web/index.html

```diff
@@ -70,287 +69,350 @@ <h3 class="sub-header">
+                                                No recipes found
+                                            </div>
                                         </div>
-                                    </div>
+                                    </transition>
                                 </div>
                             </div>
+
                             <div class="panel" :class="{ 'center-panel': currentPanelIndex === 0 }">
                                 <div class="ingredients-list">
                                     <h3>Crafting Recipe</h3>
                                     <transition name="slide-right" mode="out-in">
                                         <div :key="currentElement">
-                                            <table>
-                                                <tr v-for="recipe in craftingTree">
-                                                    <td class="item"
-                                                        :class="{'item-used': !saveData.includes(recipe.A)}"
-                                                        @click="navigateTo(recipe.A);">
-                                                        {{ data.index[recipe.A][0] }}&nbsp;{{ data.index[recipe.A][1] }}
-                                                    </td>
-                                                    <td>+</td>
-                                                    <td class="item"
-                                                        :class="{'item-used': !saveData.includes(recipe.B)}"
-                                                        @click="navigateTo(recipe.B);">
-                                                        {{ data.index[recipe.B][0] }}&nbsp;{{ data.index[recipe.B][1] }}
-                                                    </td>
-                                                    <td>=</td>
-                                                    <td class="item"
-                                                        :class="{'item-used': !saveData.includes(recipe.C)}"
-                                                        @click="navigateTo(recipe.C);">
-                                                        {{ data.index[recipe.C][0] }}&nbsp;{{ data.index[recipe.C][1] }}
-                                                    </td>
-                                                </tr>
-                                            </table>
-                                            <div v-if="craftingTree.length == 0">
-                                                <div class="item"
-                                                    :class="{'item-used': !saveData.includes(currentElement)}">
-                                                    {{ data.index[currentElement][0] }}&nbsp;{{
-                                                    data.index[currentElement][1] }}
+                                            <transition name="slide-right" mode="out-in">
```

Comment:
again, added some additional transitions to allow for more seamless stages of data loading